src/app/containers/RadioPageBlocks/Blocks/OnDemandHeading/index.jsx

```diff
@@ -49,7 +54,13 @@ const HeadingContainer = ({ idAttr, brandTitle, releaseDateTimeStamp }) => {
   });
 
   return (
-    <StyledHeadline script={script} service={service} id={idAttr} tabIndex="-1">
+    <StyledHeadline
+      script={script}
+      service={service}
+      id={idAttr}
+      tabIndex="-1"
```

Comment:
Whats the tabindex for? And the id above?

Reply:
id is how we set the `content` ID, to make skip to content work

Tab Index was pre-existing code across all radio pages - we just had a call to understand why it is set to -1 here instead of just removed entirely

Unless you have a better idea, as far as we can tell, the only difference is that with tab index set to -1, whenever the user clicks "skip to content", the element becomes highlighted.  Without the tab index, it isn't highlighted, but the user is still skipped to it

We're not sure which behavious is preferable - any suggestions?  cc @aeroplanejane @rebeccarfmcginnbbc @RichardPK

Reply:
Can you confirm the skip to content link is going to the visually hidden h1? The visually displayed strong tag with aria hidden does not need a tab index value...

Reply:
"Without the tab index, it isn't highlighted, but the user is still skipped to it successfully" FYI @ryanmccombe this doesn't work with all our supported AT without tabindex -1

Reply:
👍 cheers - we will leave the tab-index of -1 there in that case